Fixed agent failing with default config

agilov opened the pull request:
If try run agent with default config provided here
~$ /opt/td-agent-bit/bin/td-agent-bit -c //etc/td-agent-bit/td-agent-bit.conf
Fluent Bit v1.0.6
Copyright (C) Treasure Data

Error: No Input(s) have been defined. Aborting

jsonmaur commented:
https://github.com/timberio/docs/commit/6eaf18afe624257597c4427bd54d0a3907f3b6b2

agilov committed:
Fixed agent failing with default config

If try run agent with default config provided here
~$ /opt/td-agent-bit/bin/td-agent-bit -c //etc/td-agent-bit/td-agent-bit.conf
Fluent Bit v1.0.6
Copyright (C) Treasure Data

Error: No Input(s) have been defined. Aborting
